variables.tf

```diff
@@ -18,6 +18,18 @@ variable "plan_validation" {
   default     = true
 }
 
+variable "existing_kms_instance_guid" {
+  description = "The GUID of the Hyper Protect or Key Protect instance in which the key specified in var.kms_key_crn is coming from. Required if var.skip_iam_authorization_policy is false in order to create an IAM Access Policy to allow Hyper Protect or Key Protect to access the newly created Postgresql database instance."
+  type        = string
+  default     = null
+}
+
+variable "skip_iam_authorization_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of an IAM authorization policy that permits the PostgreSQL database instances in the Resource group to read the encryption key from the HPCS instance in `existing_kms_instance_guid`."
```

Comment:
Set to true to skip the creation of an IAM authorization policy that permits all PostgreSQL database instances in the provided resource group reader access to the instance specified in the `existing_kms_instance_guid` variable.